subsys/bluetooth/audio/aics_internal.h

```diff
@@ -0,0 +1,131 @@
+/** @file
+ *  @brief Internal APIs for Bluetooth AICS.
+ */
+
+/*
+ * Copyright (c) 2020 Bose Corporation
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_AICS_INTERNAL_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_AICS_INTERNAL_
+#include <zephyr/types.h>
+#include <bluetooth/gatt.h>
+
+#if defined(CONFIG_BT_AICS)
+#define BT_AICS_MAX_DESC_SIZE CONFIG_BT_AICS_MAX_INPUT_DESCRIPTION_SIZE
+#else
+#define BT_AICS_MAX_DESC_SIZE 1
+#endif /* CONFIG_BT_AICS */
+
+/* AICS opcodes */
+#define BT_AICS_OPCODE_SET_GAIN                    0x01
+#define BT_AICS_OPCODE_UNMUTE                      0x02
+#define BT_AICS_OPCODE_MUTE                        0x03
+#define BT_AICS_OPCODE_SET_MANUAL                  0x04
+#define BT_AICS_OPCODE_SET_AUTO                    0x05
+
+/* AICS status */
+#define BT_AICS_STATUS_INACTIVE                    0x00
+#define BT_AICS_STATUS_ACTIVE                      0x01
+
+#define BT_AICS_INPUT_MODE_IMMUTABLE(gain_mode) \
+	((gain_mode) == BT_AICS_MODE_MANUAL_ONLY || (gain_mode) == BT_AICS_MODE_AUTO_ONLY)
+
+#define BT_AICS_INPUT_MODE_SETTABLE(gain_mode) \
+	((gain_mode) == BT_AICS_MODE_MANUAL_ONLY || (gain_mode) == BT_AICS_MODE_MANUAL)
```

Comment:
Should this be "AUTO" || "MANUAL"?

Only three of the four modes are mentioned in these macros - the fourth mode ("AUTO") should belong in one of them.

Reply:
Yes, it was missing AUTO. Thanks!